src/modules/commands/discord/outfitstats.ts

```diff
@@ -30,7 +30,10 @@ export default new Command<discord.Message>({
 
     memberStats.outfitMember = memberStats.outfitMember.filter(
       (outfitMember) => {
-        return typeof outfitMember.character !== 'undefined'
+        return (
+          typeof outfitMember.character.stats[8] !== 'undefined' &&
+          typeof outfitMember.character !== 'undefined'
```

Comment:
The order of conditions needs to be flipped because if the second one is false, it will cause a runtime error at the first one.